src/libraries/Common/src/Interop/Windows/Ole32/Interop.IStream.COMWrappers.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+
+internal static partial class Interop
+{
+    internal static partial class Ole32
+    {
+        /// <summary>
+        /// IStream interface. <see href="https://docs.microsoft.com/en-us/windows/desktop/api/objidl/nn-objidl-istream"/>
+        /// </summary>
+        /// <remarks>
+        /// The definition in <see cref="System.Runtime.InteropServices.ComTypes"/> does not lend
```

Comment:
I'm not sure this is needed in this case. The interface is the C# projection and can be C# centric is desired. The part that should be blittable and efficient is the vtable definitions that are used to implement this interface.